formats/service_manual_guide/frontend/examples/basic_with_related_discussions.json

```diff
@@ -17,6 +17,28 @@
       { "title" : "What is it, why it works and how to do it", "href" : "#what-is-it-why-it-works-and-how-to-do-it" }
     ]
   },
+  "links": {
+    "topics": [{
+      "content_id": "f08d0fbb-c14e-4c08-8260-a24c9eacceb4",
+      "title": "Agile delivery",
+      "base_path": "/service-manual/agile-delivery",
```

Comment:
This should probably be `"/topic/service-manual/agile-delivery"`, I think (we planned that the service-manual topic would live under /topic like all the other topics).  Similarly elsewhere in this PR.